src/uv-common.h

```diff
@@ -378,6 +378,6 @@ struct uv__loop_internal_fields_s {
 # define _COMPILER_MSAN_ENABLED_
 #endif
 #endif
-void uv__msan_unpoison(const volatile void *a, size_t size);
+static inline void uv__msan_unpoison(const volatile void *a, size_t size);
```

Comment:
That only works if you put the body into the header also.

Reply:
Do you think keeping the MSAN define is useful or just move everything inside the function and that's enough?

Reply:
As is, probably just move it inside the function.